tools/api_versioning/generate_api_version_header.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+"""Parses a file containing the API version (X.Y.Z format), and outputs (toxi
+stdout) a C++ header file with the ApiVersion struct, and the value.
+"""
+import sys
+
+FILE_TEMPLATE = """#pragma once
+namespace Envoy {
+
+struct ApiVersion {
```

Comment:
Do you really want to define this datatype in a template in a Python file? :) Up to you, but it could be useful for code navigation to have somewhere other than non-generated code.

Reply:
Good idea.
Moved to a different header file.